keel-sql/src/main/resources/db/changelog/20210226-generate-event-columns.yml

```diff
@@ -0,0 +1,74 @@
+databaseChangeLog:
+- changeSet:
+    id: generate-event-timestamp
+    author: fletch
+    changes:
+    - dropIndex:
+        tableName: event
+        indexName: event_uid_timestamp_idx
+    - dropIndex:
+        tableName: event
+        indexName: event_scope_ref_timestamp_idx
+    - dropIndex:
+        tableName: event
+        indexName: event_scope_application_timestamp_idx
+    - dropColumn:
+        tableName: event
+        columnName: application
+    - sql:
+        sql: |
+          alter table event
+          add column ref_gen varchar(255) generated always as (json ->> '$.ref')
+          not null
+          after scope;
```

Comment:
I added this as a new column instead of replacing the existing one

Reply:
I have to say, I know that the rename is just for backwards compatibility, but having the `_gen` suffix communicate that this is a generated column is actually nice.

Reply:
Are you saying you like Hungarian Notation? ¬_¬